index.html

```diff
@@ -1,322 +1,373 @@
+</div>
+      <section class="about">
+        <div class="about-content">
+        <h2 class="about-heading">About me</h2>
+        <p class="about-disc">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experienc! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button type="button" class="about-btn">Get My Resume</button>
         </div>
+        <div class="about-img">
+          <div class="about-overly"></div>
 
-        <button class="form-button" type="submit" button-interaction="hover">         
-            <h3 class="form-button-text">Get in touch</h3>
-        </button>
+        </div>
+        <div class="about-line"></div>
+    </section>
+    <section class="qualification">
+      <div class="qualification-type">
+       <h3 class="qualification-heading"><i class="fa-solid fa-diamond"></i>Language</h3>
+          <ul class="qualification-list">
+            <li class="qualification-item">Javascript</li>
+            <li class="qualification-item">Ruby</li>
+            <li class="qualification-item">Html</li>
+            <li class="qualification-item">Css</li>
+          </ul>
+      </div>
+      <div class="qualification-type">
+       <h3 class="qualification-heading"><i class="fa-solid fa-square"></i>Frameworks</h3>
+          <ul class="qualification-list">
+            <li class="qualification-item">Bootstrap</li>
+            <li class="qualification-item">Ruby on Rails</li>
+            <li class="qualification-item">RSPec</li>
+            <li class="qualification-item">CapyBara</li>
+            <li class="qualification-item">Selenium</li>
+          </ul>
+      </div> 
+      <div class="qualification-type">
+       <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
+          <ul class="qualification-list">
+            <li class="qualification-item">Codekit</li>
+            <li class="qualification-item">Github</li>
+            <li class="qualification-item">Codepen</li>
+            <li class="qualification-item">Gitlab</li>
+            <li class="qualification-item">Terminal</li>
+          </ul>
+      </div>
+    </section>
+    <section class="contact">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+    <form action="https://formspree.io/f/xgebvjyw" method="POST" class="contact-form">
+      <input type="text" class="contact-input" placeholder="Full name" maxlength="30" required>
+      <input type="email" class="contact-input" placeholder="Email address" required>
+      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Enter text here"></textarea>
+      <button class="contact-btn" type="submit">Get in touch</button>
     </form>

```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design :blush: 

  - In the design, on the desktop screen we have `first name` and `last name` fields instead of `full name` field
  
  | Your Design |Figma Design |
  | :---:   | :---: | 
  | ![image](https://user-images.githubusercontent.com/92303891/228368854-a6f9af8e-adc2-48b0-9f66-96482ae2c06e.png) | ![image](https://user-images.githubusercontent.com/92303891/228368615-ae6e3751-b510-4906-9852-beb47175c223.png) |